app/assets/stylesheets/modules/_family.scss

```diff
@@ -0,0 +1,149 @@
+.family {
+  // @include span-columns(12 of 12);
+  width: 95%;
+  margin-left: 2.5%;
+
+  background-color: $white;
+  border-radius: $base-border-radius;
+  padding: em(40);
+
+  @include media($medium-screen-up) {
+    @include span-columns(10 of 12);
+    @include shift(1);
+
+    padding: em(90);
+  }
+
+  &-section-one {
+
+    &-content {
+      @include span-columns(10 0f 10);
+
+      @include media($medium-screen-up) {
+        @include span-columns(6.8 of 10);
+        margin-left: em(10);
+      }
+    }
+
+    &-image {
+      @include span-columns(12 of 12);
+      background: image-url("julian-michelle.png");
+      background-repeat: no-repeat;
+      background-size: 100%;
+      height: em(200);
+      display: block;
+
+      @include media($medium-screen-up) {
+        @include span-columns(3 of 10);
+        background-size: 100%;
+        display: inline-block;
+        height: em(120);
+      }
+
+    }
+
+  }
+
+  &-section-two-image {
+    background: image-url("sf-move.png");
+    background-repeat: no-repeat;
+    background-size: em(260);
+    display: block;
+    height: em(110);
+    width: em(260);
```

Comment:
Properties should be ordered background, background-repeat, background-size, display, height, margin, width